docs/help_out.txt

```diff
@@ -18,6 +18,10 @@ GLOBAL OPTIONS:
    --port value   Server listening port (default: 3100) [$EIGENDA_PROXY_PORT]
    --version, -v  print the version
 
+   Cert Verifier
+
+   --eigenda.cert-verification-disabled  Whether to verify certificates received from EigenDA disperser. (default: false) [$EIGENDA_PROXY_EIGENDA_CERT_VERIFICATION_DISABLED]
+
    EigenDA Client
```

Comment:
Let's add V1 Suffix to both of these categories (EigenDA Client and Cert Verifier)?

Reply:
[4e6ea09](https://github.com/Layr-Labs/eigenda-proxy/pull/344/commits/4e6ea09a7a6f352e6cc80a76ce0cb9aac41b8737)